src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -2529,11 +2530,20 @@ TEST_CASE("soroban transaction validation", "[tx][envelope][soroban]")
     resources.writeBytes = InitialSorobanNetworkConfig::TX_MAX_WRITE_BYTES;
     resources.extendedMetaDataSizeBytes =
         InitialSorobanNetworkConfig::TX_MAX_EXTENDED_META_DATA_SIZE_BYTES;
-    resources.footprint.readOnly.resize(
-        InitialSorobanNetworkConfig::TX_MAX_READ_LEDGER_ENTRIES -
-        InitialSorobanNetworkConfig::TX_MAX_WRITE_LEDGER_ENTRIES);
-    resources.footprint.readWrite.resize(
-        InitialSorobanNetworkConfig::TX_MAX_WRITE_LEDGER_ENTRIES);
+
+    auto keys =
+        LedgerTestUtils::generateValidUniqueLedgerEntryKeysWithExclusions(
+            {}, (InitialSorobanNetworkConfig::TX_MAX_READ_LEDGER_ENTRIES -
```

Comment:
Is this not just `InitialSorobanNetworkConfig::TX_MAX_READ_LEDGER_ENTRIES`?

Reply:
Oh yeah I missed this as I was moving around code. Fixed